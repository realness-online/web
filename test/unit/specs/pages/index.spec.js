import Vue from 'vue'
import { shallow } from 'vue-test-utils'
import index from '@/pages/index'

describe('index.vue', () => {
  it('renders navigation for the application', () => {
    let wrapper = shallow(index)
    expect(wrapper.element).toMatchSnapshot()
  })
})
