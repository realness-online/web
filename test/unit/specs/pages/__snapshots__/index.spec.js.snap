// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@/pages/index.vue renders navigation for the application 1`] = `
<section
  id="home"
>
  <!---->
   
  <aside>
    <!---->
     
    <!---->
  </aside>
</section>
`;
